packages/react/src/components/Popover/index.tsx

```diff
@@ -14,13 +20,47 @@ export const Popover = ({
   onClickOutside,
   borderless = false,
 }: PopoverProps) => {
+  const popoverRef = useRef<HTMLDivElement>(null);
   useHandleClickOutside(parentRef, onClickOutside);
+  const [topPosition, setTopPosition] = useState(0);
+
+  useLayoutEffect(() => {
+    const handleWindowResize = () => {
+      if (!parentRef.current || !popoverRef.current) return;
+
+      const { offsetTop: parentOffsetTop, clientHeight: parentClientHeight } =
+        parentRef.current;
+      const { clientHeight: popoverClientHeight } = popoverRef.current;
+
+      if (
+        parentOffsetTop + parentClientHeight + popoverClientHeight >
+          window.innerHeight &&
+        parentOffsetTop - popoverClientHeight > 0
```

Comment:
Can you just store this into a variable well named to help to understand quickly what is the purpose of this statement please.

Reply:
https://github.com/openfun/cunningham/pull/133/commits/c39dae425ee9a03fb0ab8aae7b6e64c34d62910e#diff-461fdff46493cb1f71f9f1f6c6c787893fc184960d53f5e288085d47b92a6fbd